src/App.js

```diff
@@ -1,74 +1,77 @@
-import React from 'react';
-import './App.css';
-import Title from './components/title/title';
-import UserNameCollection from './pages/user-name-collection';
-import UserEmailCollection from './pages/user-email-collection';
-import CongratsUser from './pages/congratulations';
+import React from "react";
+import "./App.css";
+import Title from "./components/title/Title";
+import UserNameCollection from "./pages/user-name-collection";
+import UserEmailCollection from "./pages/user-email-collection";
+import CongratsUser from "./pages/congratulations";
 
 class App extends React.Component {
-   state = {
-      step: 1,
-      email: '',
-      firstName: '',
-      lastName: ''
-   }
+    state = {
+        step: 1,
+        email: "",
+        firstName: "",
+        lastName: "",
+    };
 
-   // proceed to next step
-   nextStep = () => {
-      const { step } = this.state;
-      this.setState({
-         step: step + 1
-      });
-   }
+    // proceed to next step
+    nextStep = () => {
+        const { step } = this.state;
 
-   // handle change upon submit
-   handleChange = input => e => {
-      this.setState({ [input]: e.target.value });
-   }
+        console.log("Next");
 
-   render() {
-      const { step } = this.state;
-      const { email, firstName, lastName } = this.state;
-      const values = { email, firstName, lastName };
+        this.setState({
+            step: step + 1,
+        });
+    };
 
-      switch (step) {
-         case 1:
-            return (
-               <div className="appComponent">
-                  <Title />
-                  <UserEmailCollection
-                     nextStep={this.nextStep}
-                     handleChange={this.handleChange}
-                     values={values}
-                  />
-               </div>
-            )
+    // handle change upon submit
+    handleChange = e => {
```

Comment:
"e" is the event that you get when the input changes and already contains the name and the value of the input, so you don't need to pass the name manually on this function